cdc/sink/dispatcher/event_router.go

```diff
@@ -56,27 +57,92 @@ func (r *partitionDispatchRule) fromString(rule string) {
 // EventRouter is a router, it determines which topic and which partition
 // an event should be dispatched to.
 type EventRouter struct {
-	rules []struct {
+	defaultTopic string
+	rules        []struct {
 		partitionDispatcher partition.Dispatcher
 		topicDispatcher     topic.Dispatcher
 		filter.Filter
 	}
 }
 
+// NewEventRouter creates a new EventRouter
+func NewEventRouter(cfg *config.ReplicaConfig, defaultTopic string) (*EventRouter, error) {
```

Comment:
The check for defaultTopic is insufficient here, but it's not related to this PR, so I'll fix it in the next PR.